src/test/java/com/brew/oauth20/server/provider/authorizetype/AuthorizeTypeProviderAuthorizationCodeTest.java

```diff
@@ -44,22 +44,23 @@ private static Stream<Arguments> should_return_valid_result() {
         var clientModelFixture = new ClientModelFixture();
         var client = clientModelFixture.createRandomOne(1, new ResponseType[]{ResponseType.code});
         var url = client.redirectUriList().get(0).redirectUri();
+        var scope = client.scopeList().get(0).scope().getScope();
         return Stream.of(
-                Arguments.of(client, faker.letterify("????????????????"), url, new ValidationResultModel(true, null))
+                Arguments.of(client, faker.letterify("????????????????"), url, scope, new ValidationResultModel(true, null))
```

Comment:
[874 875 Diff](https://github.com/BrewInteractive/oauth-server/compare/TMID-875-refactor-validate-method-in-client-validator-in-o-auth-server-project...TMID-874-refactor-validate-method-in-base-authorize-type-provider-in-o-auth-server-project#diff-32ec35518a0a6b8e09acfecd13afc0128392b9e3fd125b26e6dfa0847b2c0a5eR83)
null parameter is still not clear on my side. it still exists with both pr (875, 874)